R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+      L202.maxSubResource
+
+    L202.RenewRsrcCurves %>%
+      add_title("Renewable resource curves") %>%
+      add_units("units") %>%  # TODO
+      add_comments("A_agRsrcCurves written to all regions") %>%
+      add_legacy_name("L202.RenewRsrcCurves") %>%
+      add_precursors("aglu/A_agRsrcCurves", "common/GCAM_region_names") ->
+      L202.RenewRsrcCurves
+
+    L202.UnlimitedRenewRsrcCurves %>%
+      add_title("Unlimited renewable resource curves") %>%
+      add_units("units") %>%   # TODO
+      add_comments("A_agUnlimitedRsrcCurves written to all regions") %>%
+      add_legacy_name("L202.UnlimitedRenewRsrcCurves") %>%
+      add_precursors("aglu/A_agUnlimitedRsrcCurves", "common/GCAM_region_names") ->
+      L202.UnlimitedRenewRsrcCurves
+
+    L202.UnlimitedRenewRsrcPrice %>%
+      add_title("Unlimited renewable resource price") %>%
+      add_units("1975$/kg") %>%
+      add_comments("A_agUnlimitedRsrcCurves written to all regions") %>%
+      add_legacy_name("L202.UnlimitedRenewRsrcPrice") %>%
+      add_precursors("aglu/A_agUnlimitedRsrcCurves", "common/GCAM_region_names") ->
+      L202.UnlimitedRenewRsrcPrice
+
+    L202.Supplysector_in %>%
+      add_title("Generic supplysector info for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_supplysector written to all regions") %>%
+      add_legacy_name("L202.Supplysector_in") %>%
+      add_precursors("aglu/A_an_input_supplysector", "common/GCAM_region_names") ->
+      L202.Supplysector_in
+
+    L202.SubsectorAll_in %>%
+      add_title("Generic subsector info for inputs to animal production technologies") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_subsector written to all regions") %>%
+      add_legacy_name("L202.SubsectorAll_in") %>%
+      add_precursors("aglu/A_an_input_subsector", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.SubsectorAll_in
+
+    L202.StubTech_in %>%
+      add_title("Identification of stub technologies for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology written to all regions") %>%
+      add_legacy_name("L202.StubTech_in") %>%
+      add_precursors("aglu/A_an_input_technology", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.StubTech_in
+
+    L202.StubTechInterp_in %>%
+      add_title("Generic technology info for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology written to all regions") %>%
+      add_legacy_name("L202.StubTechInterp_in") %>%
+      add_precursors("aglu/A_an_input_technology", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.StubTechInterp_in
+
+    L202.GlobalTechCoef_in %>%
+      add_title("Coefficients for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology across all base and future years") %>%
```

Comment:
(additional comment) these technologies are pass-through, used for competing different primary sources for animal feed commodities. No transformations are taking place, and the coefficients are 1 in all years.